### DISCUSSION TITLE
Create local backup of a page from archive.org?

### DISCUSSION THREAD

**Discussion by James-Firth** [ArchiveBox/ArchiveBox#1338] [Ideas]
I came across a blog post I quite liked but hadn't pulled into archivebox. Since then, the blog has gone down.

I noticed that Archive.org _happened_ to have grabbed a capture of the site before it went down. It would be great to be able to grab from them in situations like this, as ArchiveBox makes it easy to give snapshots to them.

Thoughts?

Somewhat related to: https://github.com/ArchiveBox/ArchiveBox/discussions/972

**Comment by pirate** [comment]
If you download the WARC from Archive.org, you can place it into the `warc/` folder in a snapshot dir and ArchiveBox wont delete it or mess with it, and it will show up in the UI.

Proper WARC importing is not yet implemented yet, you should follow this issue for updates: https://github.com/ArchiveBox/ArchiveBox/issues/160 where we already have a discussion about this.

**Reply by James-Firth** [reply]
Ah I didn't come across that issue, thanks I'll follow that!
